configs/ttalps_analysis/ttalps_extra_collections.py

```diff
@@ -15,13 +15,9 @@
     # 1=PFIsoVeryLoose, 2=PFIsoLoose, 3=PFIsoMedium, 4=PFIsoTight, 5=PFIsoVeryTight, 6=PFIsoVeryVeryTight
     "pfIsoId": (1, 6),
   },
-  "LooseDSAMuons": {
-    "inputCollections": ("DSAMuon",),
-    "pt": (15., 9999999.),
-    "eta": (-2.5, 2.5),
-  },
-  # "LooseMuonsAndDSAMuons": {
-  #   "inputCollections": ("Muon", "DSAMuon"),
+  # "LooseDSAMuons": {
+  #   "inputCollections": ("DSAMuon",),
+  #   "displacedId": (1, 9999990),
   #   "pt": (15., 9999999.),
   #   "eta": (-2.5, 2.5),
   # },
```

Comment:
Yes, LooseMuonsAndDSAMuons only had pt and eta cuts on Muon and DSAMuon collection, I've added AllLooseMuons in histograms instead where we use LooseMuons and LooseDSAMuons. But I included it again now in case we'd like it at some point